### DISCUSSION TITLE
not committing apiKey?

### DISCUSSION THREAD

**Discussion by SpadarShut** [lingui/js-lingui#1219] [Q&A]
My project is open-source and I would like to not commit apiKey in the .linguirc file. 
Is there a way to get the apiKey for CLI usage from the environment or somewhere else?

**Comment by MichaelHoste** [comment]
I'm not sure if there are ways to get variables from the environment in the `.linguirc`.

What I would do is to gitignore the complete `.linguirc` file and add a new identical `.linguirc.example` file without the apiKey in the repository.

**Comment by timofei-iatsenko** [comment]
linguija uses cosmicconfig to load `.linguirc` it means you can use `js` or `ts` version of this and do whatever magic you want inside.
